releasenotes/notes/fix-unicode-decode-errors-pylons-52d28b22f82e184a.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    pylons: This fix resolves an issue where str.decode caused critical unicode decode errors.
```

```suggestion
    pylons: This fix resolves an issue where ``str.decode`` caused critical unicode decode errors.
```

the `build_docs` job was failing because of spellchecker. if we make this an inline code block the spellchecker will ignore it.

the alternative is to add `str` to our spelling word list. (inline code block seemed to make sense for this specific instance)